sample.craft-cli.php

```diff
@@ -74,6 +74,18 @@
     'environment' => null,
 
     /**
+     * Dotenv path
+     *
+     * (optional) If using phpdotenv, specify the path to
+     * the directory containing your .env file, and
+     * craft-cli load with load environment variables
+     * from it
+     */
+
+    // 'dotenv_path' => __DIR__,
```

Comment:
I prefer the `__DIR__` default. Can you restore this? I think it'd be better to have the default, and then do a `file_exists` before loading Dotenv.